### PR TITLE
feat(endpoint): support the movies endpoint

### PR DESCRIPTION
**Pull request by tzaitsev**
Add the ability to post to /movies.

### CODE REVIEW

File: lib/plugins/features/movies/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  }]);
+
+  next();
+
+};
+
+exports.register.attributes = {
+  name: 'movies'
+};
+
```
Comment (kylekwong) [inline lib/plugins/features/movies/index.js]: I think this is an extraneous newline

**Review by kylekwong — APPROVED** [review]

File: lib/plugins/features/movies/index.js (hunk 1 above)
Comment (tzaitsev) [reply lib/plugins/features/movies/index.js]: Done.

Comment (ajpawlicki) [reply lib/plugins/features/movies/index.js]: Don't we usually add newlines at the end of each file?

**Review by ajpawlicki — APPROVED** [review]

**Comment by ajpawlicki** [comment]
Nit: we usually add a brief what and why to the description

File: lib/plugins/features/movies/index.js (hunk 1 above)
Comment (kylekwong) [reply lib/plugins/features/movies/index.js]: Yes but there were two

**Pull request merged**